tests/testthat/test-stats-mlm.R

```diff
@@ -0,0 +1,9 @@
+context("stats-mlm")
+
+fit_mlm <- lm(cbind(mpg, disp) ~ wt, mtcars)
+df_tidy <- tidy(fit_mlm, conf.int = TRUE)
+
+test_that("tidy.mlm works", {
+  testthat::expect_equal(dim(df_tidy), c(4L, 8L))
```

Comment:
Can you use `modeltests::check_tidy_output()` and `modeltests::check_dims()` here for consistency with the other tests?